batchspawner/batchspawner.py

```diff
@@ -334,7 +335,7 @@ def start(self):
         if self.user and self.user.server and self.user.server.port:
             self.port = self.user.server.port
             self.db.commit()
-        elif (jupyterhub.version_info < (0,7) and not self.user.server.port)  or \
+        elif (jupyterhub.version_info < (0,7) and not self.user.server.port) or \
```

Comment:
Hi @yarikoptic, In general, we discourage PRs that are primarily style changes. In this case, I'm happy to be rid of the `\`. You may as well go ahead and eliminate the `\` here by placing outer parens on both conditionals.

Reply:
Alternatively:
```
elif (jupyterhub.version_info < (0,7) and not self.user.server.port) or (
    jupyterhub.version_info >= (0,7) and not self.port
):
```